### PR TITLE
Thomas/feature/8

### PR DESCRIPTION
**Pull request by Unaimend**
handles several cases related to missing internet connection as well as some cases where just a subset of DB's could be downloaded.

Please discuss how I handle the cases i.e. returning empty AnnotationResults or should we abort? 
Further, there were two cases where I was not sure what to return. Those are marked in the code by `NotImplemented` errors.

This PR also breaks several test cases that I did not fix because we did not decide how to handle missing DBs in final

### CODE REVIEW

File: src/annotateBiGG.py [hunk 1]
```diff
@@ -24,8 +24,13 @@ def annotateBiGG_entry(entry:str,  database:pd.DataFrame = pd.DataFrame()) -> tu
...
         # load the database
         config = get_config()
-        db_path =  os.path.join(get_database_path(), config["databases"]["BiGG"]["file"])
-        bigg = pd.read_table(db_path)
+        try:
+          db_path =  os.path.join(get_database_path(), config["databases"]["BiGG"]["file"])
+          bigg = pd.read_table(db_path)
+        except FileNotFoundError as e:
+          print(f"File could not be found {e}")
+          # TODO What do we return
```
Comment (Unaimend) [inline src/annotateBiGG.py:32]: @Porthmeus

File: src/annotateModelSEED.py [hunk 2]
```diff
@@ -142,11 +144,17 @@ def annotateModelSEED_entry(entry:str,  database:pd.DataFrame = pd.DataFrame())
...
     if len(database) == 0:
         # load the database
         config = get_config()
         db_path =  os.path.join(get_database_path(), config["databases"]["ModelSeed"]["file"])
-        mseed = pd.read_table(db_path, low_memory= False)
+        try:
+          mseed = pd.read_table(db_path, low_memory= False)
+        except FileNotFoundError as e:
+          print(f"No such file {e}")
+          raise NotImplementedError()
```
Comment (Unaimend) [inline src/annotateModelSEED.py:156]: @Porthmeus

**Comment by Unaimend** [comment]
image

Still broken

**Comment by Unaimend** [comment]
> image
> 
> Still broken
Fixed by 
3df8a33a390c8df1b1b5779954ee22f10f292985

File: tests/test_PubchemInfo.py [hunk 3]
```diff
@@ -20,5 +20,9 @@ def test_looksLikeSumFormula(self):
     def test_getPubchemInfo(self):
         self.assertIsInstance(getPubchemInfo(["alkdjfa", "Carnitine"]), pd.DataFrame)
 
+    #def test_getPubchemInfo_noInternet(self):
```
Comment (Unaimend) [inline tests/test_PubchemInfo.py:23]: This test can only be used when there is no internet connect

File: src/annotateBiGG.py (hunk 1 above)
Comment (Porthmeus) [reply src/annotateBiGG.py]: fixed in 6378b0b

File: src/annotateModelSEED.py (hunk 2 above)
Comment (Porthmeus) [reply src/annotateModelSEED.py]: fixed in 6378b0b

**Comment by Unaimend** [comment]
Also fixes #103

**Review by Porthmeus — APPROVED** [review]

**Pull request merged**